include/zephyr/drivers/timer/ti_j721e_dmtimer.h

```diff
@@ -0,0 +1,159 @@
+/* Copyright (C) 2023 BeagleBoard.org Foundation
+ * Copyright (C) 2023 S Prashanth
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_TIMERS_TI_J721E_DMTIMER_H_
+#define ZEPHYR_DRIVERS_TIMERS_TI_J721E_DMTIMER_H_
+
+#include <zephyr/devicetree.h>
+#include <zephyr/dt-bindings/interrupt-controller/arm-vim.h>
+
```

Comment:
Is this needed? looks like this is from a PR that is not merged yet, please provide context for reviewers if there a dependencies like this.